com.unity.render-pipelines.high-definition/Runtime/Material/Decal/DecalUtilities.hlsl

```diff
@@ -261,3 +261,25 @@ DecalSurfaceData GetDecalSurfaceData(PositionInputs posInput, float3 vtxNormal,
 
     return decalSurfaceData;
 }
+
+DecalSurfaceData GetDecalSurfaceData(PositionInputs posInput, FragInputs input, inout float alpha)
```

Comment:
I created a new function to avoid a prototype change on GetDecalSurfaceData, but is it neccessary and should it be written somewhere in the doc ?